CONSTITUTION.md

```diff
@@ -55,11 +55,19 @@ Membership shall be open to all Union members, and to all other persons satisfyi
 
 - 4.2.5 Annual subscription is dated for the calendar year, expiring on the 31st of December of that year.
 
-### 4.3 Disqualification
+### 4.3 Expulsion
 
-- 4.3.1 A person being one month in arrears in annual subscription shall cease to be a member.
+- 4.3.1 Any member whose actions either bring discredit on the Society, who has persistently refused to comply with the rules and regulations of the Society, or who has acted in a manner prejudicial to the interests of the Society may be considered for expulsion by the executive committee.
 
-- 4.3.2 Any member whose actions either bring discredit on the Society, who has persistently refused to comply with the rules and regulations of the Society, or who has acted in a manner prejudicial to the interests of the Society may, at the discretion of the Executive Committee after an affirmative vote of sixty-six percent or more of a fully attended Executive Committee meeting be asked to appear before the Executive Committee to show cause why they should not be expelled from the Society. Should the subject member not respond to the summons, the Executive Committee may decide the matter "in absentia". Should the subject member be a member also of the Executive Committee, their place on the Executive Committee shall be taken at the direction of the President or Vice-President, by another member of the Society, not already a member of the Executive Committee, elected by the remaining members of the Executive Committee for the duration of that discussion of this rule. The Subject member shall have the right of appeal against the action of the Executive Committee. Such an appeal must be lodged with the President or the Secretary by the subject member within fourteen days of his notification of the Executive Committee's decision. The appeal will be considered at a Special General Meeting. Persons expelled from the Society under this rule will be given the option of resigning. No person who has been expelled from the Society under this rule (whether by expulsion or opted resignation) shall be re-admitted to membership except by consent of a majority of attending members at a General Meeting.
+- 4.3.2 The decision to expell a member must be conducted by a fully attended Executive Committee meeting with an affirmative vote of sixty-six percent or more.
+
+- 4.3.3 A member who is being considered for expulsion shall have the right to to appear before the Executive Committee to show cause why they should not be expelled from the Society.
+
+- 4.3.4 Should the subject member be a member also of the Executive Committee, their place on the Executive Committee shall be taken at the direction of the President or Vice-President, by a General Committee member, elected by the remaining members of the Executive Committee for the duration of that discussion of this rule.
```

Comment:
Just note that in the unlikely case that the motion to replace 'Liaison Officer' with 'General Committee' doesn't pass, this will need to read 'Liaison Officer.